src/main/java/com/google/devtools/build/lib/analysis/RunfilesSupport.java

```diff
@@ -73,6 +73,8 @@
 @AutoCodec
```

Comment:
All changes in this file are refactorings that are independent of the business logic change.

Could you create a separate PR just for refactorings? That'd make the logic-changing parts easier to review.